packages/alice_objectbox/lib/model/cached_alice_http_call.dart

```diff
@@ -110,4 +112,24 @@ class CachedAliceHttpCall implements AliceHttpCall {
     this.response = response;
     loading = false;
   }
+
+  @override
+  List<Object?> get props =>  [
```

Comment:
Any reason this needs to be overridden here?

Reply:
![image](https://github.com/jhomlala/alice/assets/1329033/4c491f86-9d64-4de2-845f-ed25f0175b7b)

Reply:
Oh I see, because it's an implementation of the class. Ugh. Nasty.